packages/appsync-modelgen-plugin/src/visitors/appsync-dart-visitor.ts

```diff
@@ -108,16 +118,16 @@ export class AppSyncModelDartVisitor<
       .implements([`${LOADER_CLASS_NAME}Interface`])
       .addClassMember('version', 'String', `"${this.computeVersion()}"`, undefined, ['override'])
       .addClassMember('modelSchemas', 'List<ModelSchema>', `[${modelNames.map(m => `${m}.schema`).join(', ')}]`, undefined, ['override'])
+      .addClassMember('customTypeSchemas', 'List<ModelSchema>', `[${nonModelNames.map(nm => `${nm}.schema`).join(', ')}]`, undefined, ['override'])
```

Comment:
This field now needs to be conditionally inserted for amplify-flutter version 0.3.0 and above.
(For preview release the version check may need to contain 0.3.0.rc.2)